packages/ui/src/components/atoms/LeanBoxContainer.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+
+interface StyledLeanBoxContainerProps {
+  padding?: string;
+}
+
+export const StyledLeanBoxContainer = styled.div<StyledLeanBoxContainerProps>`
+  display: flex;
+  padding: ${({ padding }) => padding ?? '16px 40px 32px 0px'};
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 8px;
+  align-self: stretch;
+`;
```

Comment:
Just a `Flex` component can be used here. Also, Don't prefix components with `Styled`

Reply:
Styled is removed